SpatialGDK/Documentation/content/get-started/get-started.md

```diff
@@ -0,0 +1,24 @@
+# Get started: Introduction
+
+We recommend you start by following the [Installation and setup guide]({{urlRoot}}/setup-and-installing.md) to set up the GDK, its Unreal Engine fork, and our Starter Project. 
```

Comment:
This is the installation and set up guide! :-)
Recommend:

"We recommend you start by following this Get started guide to get set up with the GDK, its Unreal Engine fork, and our Starter Project. "

Reply:
oops ! no link then